packages/hyperflux/functions/ReactorFunctions.tsx

```diff
@@ -113,6 +115,15 @@ export function startReactor(Reactor: React.FC): ReactorRoot {
     isRunning: false,
     Reactor,
     promise: null! as Promise<void>,
+    forceRender: () => {
+      if (!reactorRoot.isRunning) return reactorRoot.run()
+      ReactorReconciler.updateContainer(
+        <ReactorRootContext.Provider value={reactorRoot}>
+          <Reactor />
+        </ReactorRootContext.Provider>,
+        fiberRoot
+      )
+    },
     run() {
```

Comment:
let's just add a `force` flag parameter here to override the `isRunning` check, and remove the `forceRender` function